gallery/src/components/NxStatefulDropdownIconOnly/NxStatefulDropdownIconOnlyPage.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxCode, NxP, NxTextLink, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+
+import {GalleryDescriptionTile, GalleryExampleTile} from '../../gallery-components/GalleryTiles';
+
+import NxStatefulDropdownIconOnlyExample from './NxStatefulDropdownIconOnlyExample';
+
+const nxStatefulDropdownIconOnlyExampleCode = require('./NxStatefulDropdownIconOnlyExample?raw');
+
+const NxStatefulDropdownPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A stateful variation of the dropdown menu component which uses a single icon as its toggle.
+        <NxCode>NxStatefulDropdownIconOnly</NxCode> shares all of the same props as{' '}
+        <NxTextLink href="#/pages/NxStatefulDropdown">NxStatefulDropdown</NxTextLink>{' '}
```

Comment:
Is it simpler to describe it in comparison to `NxStatefulDropdown` or in comparison to `NxDropdownIconOnly`?

Reply:
> Is it simpler to describe it in comparison to `NxStatefulDropdown` or in comparison to `NxDropdownIconOnly`?

Possibly, it's just a small wording tweak: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/4a5197120b4af4c9d0e980b18077c67285f57cfb

Reply:
> Possibly, it's just a small wording tweak: 4a51971

Oh, no I was referring to the next sentence after the one you changed and to the props table.

Reply:
> > Possibly, it's just a small wording tweak: [4a51971](https://github.com/sonatype/sonatype-react-shared-components/commit/4a5197120b4af4c9d0e980b18077c67285f57cfb)
> 
> Oh, no I was referring to the next sentence after the one you changed and to the props table.

Oh, I went the wrong direction, how's this: https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/4a642b93d1d8072fa71a07e95b1f2a55335158f0